preppal-be/models/recipe.ts

```diff
@@ -1,16 +1,22 @@
-import { UUID } from "mongodb";
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
+const date = new Date().toString();
+
 const RecipeSchema = new Schema({
```

Comment:
should we have something for images?

Reply:
added image to schema, it will take the url address.